prisma/mappers.ts

```diff
@@ -0,0 +1,189 @@
+
+export function mapCampusArea(area: RawCampusArea): CampusArea {
+  // Only consider descrshort for mapping
+  switch (area.descrshort) {
+    case 'East':
+      return CampusArea.EAST;
+    case 'West':
+      return CampusArea.WEST;
+    case 'North':
+      return CampusArea.NORTH;
+    case 'Central':
+      return CampusArea.CENTRAL;
+    case 'Collegetown':
+      return CampusArea.COLLEGETOWN;
+    case 'South':
+      return CampusArea.SOUTH;
+    default:
+      throw new Error(`Unknown campus area: ${area.descrshort}`);
+  }
+}
+
+export function mapPaymentMethod(method: RawPayMethod): PaymentMethod {
+  // Only consider descrshort for mapping
+  switch (method.descrshort) {
+    case 'Meal Plan - Swipe':
+      return PaymentMethod.MEAL_SWIPE;
+    case 'Meal Plan - Debit':
+    case 'Cornell Card':
+      return PaymentMethod.BRB;
+    case 'Major Credit Cards':
+    case 'Mobile Payments':
+      return PaymentMethod.CARD;
+    case 'Cash':
+      return PaymentMethod.CASH;
+    case 'Free':
+      return PaymentMethod.FREE;
+    default:
+      throw new Error(`Unknown payment method: ${method.descrshort}`);
+  }
+}
+
+export function mapEateryType(type: RawEateryType): EateryType {
+  // Only consider descr for mapping
+  switch (type.descr) {
+    case 'Dining Room':
+      return EateryType.DINING_ROOM;
+    case 'Cafe':
+        return EateryType.CAFE;
+    case 'Coffee Shop':
+      return EateryType.COFFEE_SHOP;
+    case 'Food Court':
+      return EateryType.FOOD_COURT;
+    case 'Convenience Store':
+      return EateryType.CONVENIENCE_STORE;
+    case 'Cart':
+      return EateryType.CART;
+    case 'General':
+      return EateryType.GENERAL;
+    default:
+      throw new Error(`Unknown eatery type: ${type.descr}`);
+    }
+}
+
+export function mapEventType(eventDescription: string): EventType {
+  switch (eventDescription) {
+    case 'Available All Day':
+      return EventType.AVAILABLE_ALL_DAY;
+    case 'Breakfast':
+      return EventType.BREAKFAST;
+    case 'Brunch':
+      return EventType.BRUNCH;
+    case 'Dinner':
+      return EventType.DINNER;
+    case '':
+      // Could be empty string in the case of general events (e.g. Amit Bhatia Libe Cafe since it serves all day)
+      return EventType.EMPTY;
+    case 'Late Lunch':
+      return EventType.LATE_LUNCH;
+    case 'Lunch':
+      return EventType.LUNCH;
+    case 'Open':
+      return EventType.OPEN;
+    case 'Pants':
+      return EventType.PANTS;
+    case 'General':
+      return EventType.GENERAL;
+    case 'Free Food':
+      return EventType.GENERAL;
+    default:
+      throw new Error(`Unknown event type: ${eventDescription}`);
+  }
+}
+
+// Mapping from cornellId to Digital Ocean Spaces image URL
+const CORNELL_ID_TO_IMAGE_URL: Record<number, string> = {
+  31: 'https://appdev-upload.nyc3.digitaloceanspaces.com/eatery-images/104-West.jpg',
+  7: 'https://appdev-upload.nyc3.digitaloceanspaces.com/eatery-images/Amit-Bhatia-Libe-Cafe.jpg',
```

Comment:
All of these images have the same base url up until the eatery-images/
Store the base url somewhere (in a constants file in the src directory probably), and then concatenate it with the unique image names in the mapImageUrl function